README.md

```diff
@@ -14,6 +14,11 @@ The documentation in the subdirectories, of the specific exporters, however,
 are intended to be purely _descriptive_. They should reflect exactly what the current version of the exporter 
 actually does, even if it doesn't exactly follow the guidelines. 
 
+### Existing available exporter repositories
+
+We have repositories for several New Relic exporters already. The full table can be found at the docs.newrelic.com site.  It 
```

Comment:
It would be better if we have a URL that points right at the page. Is that possible?

Reply:
added!